README.md

```diff
@@ -0,0 +1,705 @@
+
+> The job is skipped if image type is not 'default' or 'minimal'
+
+#### Step: Checkout *container-images*, branch 'docker-library'
+
+Checkouts *container-images* into branch 'docker-library'. The repository directory is located at `/home/runner/work/container-images/container-images`.
+The [actions/checkout@v4](https://github.com/actions/checkout/) is used.
+
+#### Step: Download changed Containerfiles
+
+Uses [actions/download-artifact@v4](https://github.com/actions/download-artifact) to download multiple (`merge-multiple: true`) artifacts with changed Containerfiles. The files are saved into the `Containerfiles/version_major/` directory.
+
+#### Step: Commit and push *Containerfiles/version_major/Containerfile.** changes
+
+> The step is skipped if '*Push to production registries*' is not checked (`inputs.production` set to `false`.)
+
+Uses [EndBug/add-and-commit@v9](https://github.com/marketplace/actions/add-commit) to commit and push Containerfiles, which was downloaded on the previous step, and changed by the previous job.
+
+The commit message is:
+```yaml
+AlmaLinux ${{ inputs.version_major }} image build as of ${{ needs.build.outputs.date_stamp }} (with ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}).
+```
+It includes AlmaLinux version major, image build date, and reference to this Github Action.
+
+## Publish images to the Docker Library
+
+Tree illustration of the workflow Jobs and Steps for AlmaLinux 9 minimal image:
+```
+Publish images to the Docker Library
+│
+├── 8 default definition preparing
+.
+.
+.
+├── 8 minimal definition preparing
+.
+.
+.
+├── 9 default definition preparing
+.
+.
+.
+├── 9 minimal definition preparing
+│   ├── Set up job
+│   ├── Checkout container-images, branch 'docker-library'
+│   ├── Checkout official-images, branch 'master'
+│   ├── Get need data for the definition
+│   ├── Render the definition
+│   ├── Upload the definition for 9 minimal
+│   ├── Post Checkout official-images, branch 'master'
+│   ├── Post Checkout container-images, branch 'docker-library'
+│   └── Complete job
+│
+└── Create Pull Request with the new definition file
+    ├── Set up job
+    ├── Checkout official-images, branch 'master'
+    ├── Sync official-images with docker-library/official-images
+    ├── Download all definitions
+    ├── Create head of official-images/library/almalinux
+    ├── Merge definitions into official-images/library/almalinux
+    ├── Prepare date stamp
+    ├── Prepare time stamp
+    ├── Commit and push official-images/library/almalinux
+    ├── Create Pull Request for official-images/library/almalinux
+    ├── Post Checkout official-images, branch 'master'
+    └── Complete job
+```
+
+### Inputs
+
+The workflows inputs are:
+- `pr` - boolean *'Publish to the Docker Official Library'* with the default value `true` (checked). The input indicates whether to create Pull Request to Docker with AlmaLinux *Definition File*.
```

Comment:
- `pr` - boolean *'Publish to the Docker Official Library'* with the default value `true` (checked). The input indicates whether to create a Pull Request to Docker with AlmaLinux *Definition File*.